tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -1503,35 +1626,85 @@ Feature: List a sharing permissions
+                          "const": ""
+                        },
+                        "@libre.graph.quickLink": {
+                          "const": false
+                        },
+                        "preventsDownload": {
+                          "const": false
+                        },
+                        "type": {
+                          "const": "view"
+                        },
+                        "webUrl": {
+                          "type": "string",
+                          "pattern": "^%base_url%\/s\/[a-zA-Z]{15}$"
+                        }
+                      }
+                    }
+                  }
                 }
               ]
             }
           }
         }
       }
       """
+    Examples:
+      | permissions-role |
+      | Space Viewer     |
+      | Space Editor     |
+      | Manager          |
 
-  @issues-8331
-  Scenario: user sends share invitation with all allowed roles for a folder in project space
+
+  Scenario: user sends share invitation with all allowed roles for a project space using root endpoint
     Given using spaces DAV path
     And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
     And user "Alice" has created a space "new-space" with the default quota using the Graph API
-    And user "Alice" has created a folder "folder" in space "new-space"
     And user "Brian" has been created with default attributes and without skeleton files
-    When user "Alice" gets permissions list for folder "folder" of the space "new-space" using the Graph API
+    When user "Alice" lists the permissions of space "new-space" using root endpoint of the Graph API
     Then the HTTP status code should be "200"
-    And user "Alice" should be able to send share the following invitation with all allowed permission roles
-      | resource  | folder    |
+    And user "Alice" should be able to send share the following invitation with all allowed permission roles using root endpoint using the Graph API
       | space     | new-space |
       | sharee    | Brian     |
       | shareType | user      |
 
 
-  Scenario: try to list the permissions of other user's personal space
+  Scenario: try to list the permissions of other user's personal space using root endpoint
     Given using spaces DAV path
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
     And user "Brian" has been created with default attributes and without skeleton files
-    When user "Brian" tries to list the permissions of space "Personal" owned by "Alice" using permissions endpoint of the Graph API
-    Then the HTTP status code should be "404"
+    When user "Brian" tries to list the permissions of space "new-space" owned by "Alice" using root endpoint of the Graph API
+    Then the HTTP status code should be "500"
```

Comment:
there should be an issue for this. We should get 404 like with permissions endpoint

Reply:
ok i removed that scenario from this PR and cover after bug fixed

Reply:
please, create an issue for 500. I didn't see the issue